config.json

```diff
@@ -1337,6 +1338,6 @@
       "zsh-users/zsh-syntax-highlighting",
       "zulip/zulip"
     ],
-    "labels": ["good first issue", "help wanted"]
+    "labels": ["good first issue", "help wanted", "Good first issue"]
```

Comment:
What is this one for?

Reply:
This is to follow the naming convention on the repository.